core/src/org/apache/cloudstack/storage/to/PrimaryDataStoreTO.java

```diff
@@ -51,6 +50,7 @@
     private final String url;
     private Map<String, String> details;
     private static final String pathSeparator = "/";
+    private boolean isManaged;
```

Comment:
Since no mutator is exposed and the value gets set in the constructor, considering making it ``final``.